dev/k3d-clean

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+k3d cluster list -o json | \
+    jq -r '.[].name' | \
+    xargs -I{} k3d cluster delete {}
+
+ids=$(docker ps -a --filter name=k3d --format "{{.ID}}")
+for id in $ids; do
+    docker stop "$id"
+    docker rm "$id"
+done
+
+ids=$(docker ps -a --filter name=act --format "{{.ID}}")
```

Comment:
Maybe we can put `act` in the script's name, I was surprised that it deletes something else beside k3d.

Reply:
Yes, it deletes something else as well. If `act` is used to create an environment, it creates a container to use which executes all those steps defined in the workflow files. 

If there is an issue while setting the environment up (means, an issue in the workflow files), the container is not being removed automatically. It may even be reused, which can cause issues. Those are shortly explained in the [pull requests that extends the `dev/README.md` with information about how to use `act`](https://github.com/rancher/fleet/pull/1765/files#diff-e14025c0fa40d4857e4b40fc96ea5ee995afe300014626b68cb55a479fa5b8fcR277-R288).

I'll change the name to `k3d-act-clean`.